library/src/G4LatticeReader.cc

```diff
@@ -392,12 +395,29 @@ G4bool G4LatticeReader::ProcessEulerAngles(const G4String& name) {
     G4cerr << "G4LatticeReader: Unknown rotation matrix " << name << G4endl;
     return false;
   }
-
+    
   G4double degOrRad = ProcessUnits("Angle");
   pLattice->AddValley(phi*degOrRad, theta*degOrRad, psi*degOrRad);
```

Comment:
We need to be doing the antivalley here as well.  If the user's config.txt is using the old "valley" input, then we still need to compute the antivalley for them.  You shouldn't do it by modifying that AddValley function internally; instead, you should apply the appropriate change to the Euler angles (psi -> -psi?  Or if psi==0, something else?) and call AddValley a second time with the modified values.

Reply:
Perfect, I'll add the anti-valleys there!